source/index.html.haml

```diff
@@ -5,34 +5,35 @@ title: Hey, I'm Amber Aultman
 %header
     %nav.home
     %section.intro
-        %h1 
-            Hey, 
+        %h1
+            Hey,
             %br
             I'm Amber.
-        %p{:onclick => "showHide()"} 
+        %button{:onclick => "showHide()"}
             I lead strategy and teams to build meaningful, digital interactions.
             %ion-icon{:name => "chevron-down-outline"}
     %section#about
-        %img{:src => "https://via.placeholder.com/600x600"}
-        %article
-            .bio
-                %p.large 
-                    I’m Amber Aultman, a design leader currently living in Phoenix, AZ. I’m the Head of Design at 
-                    = link_to 'Tuft & Needle', 'https://www.tuftandneedle.com/', :target => "_blank"
-                    where I'm responsible for the visual experience of our brand: everything from product packaging and retail signage to our internal apps, e-commerce platform, and other digital channels. I’m crazy passionate about mentoring my team and making sure the customer experience stays paramount as we transistion through a merger. I’m also the past President of 
-                    = link_to 'AIGA Jacksonville', 'https://jacksonville.aiga.org/', :target => "_blank"
-                    and a former teacher at 
-                    = link_to 'Bloc.io.', 'https://www.bloc.io/', :target => "_blank"
-                %p.large
-                    I’m currently accepting speaking and workshop facilitation engagements both locally and abroad. 
-                = link_to 'Get in touch', 'mailto:amber.aultman@gmail.com', :class => "button-primary"
-            .links
-                = link_to 'LinkedIn', 'https://www.linkedin.com/in/amberaultman/', :target => "_blank", :class => "button-tertiary"
-                = link_to 'Medium', 'https://medium.com/@amberaultman', :target => "_blank", :class => "button-tertiary"
-                = link_to 'Github', 'https://github.com/amberaultman', :target => "_blank", :class => "button-tertiary"
-                = link_to 'Email', 'mailto:amber.aultman@gmail.com', :class => "button-tertiary"
-            .close
-                %ion-icon{:name => "close-outline"}
+        .wrapper
```

Comment:
This looks like a huge overhaul to this content block, but really, the only change is the addition of this `wrapper` div. Instead of applying the 4% padding to the entire section, we're applying that 4% padding and grid layout to the inner wrapper div. 

This way, we can easily handle the accordion style open/close transition without any of this content displaying when it should be hidden

Reply:
Oh nice! So smart.